index.d.ts

```diff
@@ -0,0 +1,59 @@
+import { ViewStyle, LayoutAnimationConfig } from "react-native";
+import { ReactNode } from "react";
+
+interface KeyboardAccessoryViewProps {
+  style?: ViewStyle;
```

Comment:
It's better to use `StyleProp` here, why?
check this -> [link](https://spin.atomicobject.com/2018/06/02/custom-components-react-native/)